packages/api-kit/src/utils/signDelegate.ts

```diff
@@ -1,10 +1,14 @@
-import { Signer } from 'ethers'
+import { Chain, LocalAccount, Transport, WalletClient } from 'viem'
 
-export async function signDelegate(signer: Signer, delegateAddress: string, chainId: bigint) {
+export async function signDelegate(
+  walletClient: WalletClient<Transport, Chain, LocalAccount>,
+  delegateAddress: string,
+  chainId: bigint
+) {
   const domain = {
     name: 'Safe Transaction Service',
     version: '1.0',
-    chainId: chainId
+    chainId: Number(chainId)
```

Comment:
Was this failing before?

Reply:
No with ethers it was not failing, but viem's `signTypedData` function expects this value as number, instead of bigint.